tardis/montecarlo/montecarlo_numba/tests/test_vpacket.py

```diff
@@ -66,7 +66,8 @@ def test_trace_vpacket_within_shell(
     )
 
     npt.assert_almost_equal(tau_trace_combined, 8164850.891288479)
-    npt.assert_almost_equal(distance_boundary, 843684056256104.1)
+    # changed from almost equal to allclose. Now seems to work.
```

Comment:
I assume you don't know why

Reply:
no - you think it's fine?

Reply:
Presumably it is related to the data types. The comment is fine anyway